mp4/emsg.go

```diff
@@ -36,8 +38,9 @@ func DecodeEmsgSR(hdr BoxHeader, startPos uint64, sr bits.SliceReader) (Box, err
 	versionAndFlags := sr.ReadUint32()
 	version := byte(versionAndFlags >> 24)
 	b := &EmsgBox{
-		Version: version,
-		Flags:   versionAndFlags & flagsMask,
+		Version:     version,
+		Flags:       versionAndFlags & flagsMask,
+		MessageData: make([]byte, 0),
```

Comment:
@tobbee comparing to your proposal: I initialize the `MessageData` field here

Reply:
It's OK to initialize it, but it should be initialized to **nil**, which is the default zero value.
Otherwise, you make an unecessary allocation on heap, and that should be avoided in Go.
A nil-value for a slice works with both len() and append().

Reply:
Ok, changed accordingly.